scripts/gas-refund-program/staking/safety-module-stakes-tracker.ts

```diff
@@ -0,0 +1,449 @@
+import BigNumber from 'bignumber.js';
+import { BigNumber as EthersBN, CallOverrides, Contract, Event } from 'ethers';
+import { assert } from 'ts-essentials';
+import { CHAIN_ID_MAINNET, PSP_ADDRESS } from '../../../src/lib/constants';
+import { Provider } from '../../../src/lib/provider';
+import * as ERC20ABI from '../../../src/lib/abi/erc20.abi.json';
+import * as BVaultABI from './balancer-vault-abi.json';
+import { getTokenHolders } from './covalent';
+import { fetchBlockTimestampForEvents, ZERO_BN } from '../utils';
+import { getLatestTransactionTimestamp } from '../persistance/db-persistance';
+import { EpochInfo } from '../../../src/lib/epoch-info';
+import { GasRefundSafetyModuleStartEpoch } from '../../../src/lib/gas-refund';
+import { OFFSET_CALC_TIME } from '../common';
+import { BlockInfo } from '../../../src/lib/block-info';
+import {
+  reduceTimeSeries,
+  TimeSeries,
+  timeseriesComparator,
+} from '../timeseries';
+
+const SafetyModuleAddress = '0xc8dc2ec5f5e02be8b37a8444a1931f02374a17ab';
+const BalancerVaultAddress = '0xba12222222228d8ba445958a75a0704d566bf2c8';
+const Balancer_80PSP_20WETH_poolId =
+  '0xcb0e14e96f2cefa8550ad8e4aea344f211e5061d00020000000000000000011a';
+const Balancer_80PSP_20WETH_address = Balancer_80PSP_20WETH_poolId.substring(
+  0,
+  42,
+); // or 0xcb0e14e96f2cefa8550ad8e4aea344f211e5061d
```

Comment:
```suggestion
const Balancer_80PSP_20WETH_address = '0xcb0e14e96f2cefa8550ad8e4aea344f211e5061d'
const Balancer_80PSP_20WETH_poolId = `${Balancer_80PSP_20WETH_address}00020000000000000000011a`
```

maybe this'll be a bit more intuitive. Or maybe not